File: FiveTeamTest/src/com/office/test/DevelopperOffice.java
package com.office.test;
//Classe de bureau de développeur

public class DevelopperOffice extends Office {

	/**
	 * Initialise Developpeur office
	 * @param networkSocket_nb
	 * @param sectorSocket_nb
	 * @param phoneSocket_nb
	 * @param chairs_nb
	 * @param tables_nb
	 * @param person_nb
	 */
	public DevelopperOffice(int networkSocket_nb, int sectorSocket_nb,
			int phoneSocket_nb, int chairs_nb, int tables_nb) {
		super(networkSocket_nb, sectorSocket_nb, phoneSocket_nb, chairs_nb, tables_nb);
		
	}
	public void addPerson(int i) {
		super.addPerson(i);
		
	}
	// calcul free space in developper office
	public int freeSpace(){ 
    	return (int) (Person_nb - 3*networkSocket_nb+Person_nb-3*sectorSocket_nb+Person_nb-phoneSocket_nb+Person_nb-1.5*chairs_nb+Person_nb-tables_nb);
    }
	
}

File: FiveTeam_Test_php/DevelopperOffice.php
<?php


class DevelopperOffice extends Office 
{
    function __construct($networkSocket_nb, $sectorSocket_nb, $phoneSocket_nb, $chairs_nb, $tables_nb){
        parent:: __construct($networkSocket_nb, $sectorSocket_nb, $phoneSocket_nb, $chairs_nb, $tables_nb);
    }
    // calcul free space in Developer office
    public function freeSpace(){
        return (int) ($this->Person_nb - 3*$this->networkSocket_nb+$this->Person_nb-3*$this->sectorSocket_nb+$this->Person_nb-$this->phoneSocket_nb+$this->Person_nb-1.5*$this->chairs_nb+$this->Person_nb-$this->tables_nb);
    }
}